applications/nrf_desktop/configuration/nrf54h20dk_nrf54h20_cpuapp/prj.conf

```diff
@@ -0,0 +1,116 @@
+#
+# Copyright (c) 2024 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+################################################################################
+# Application Configuration
+
+CONFIG_DESKTOP_INIT_LOG_MOTION_EVENT=n
+CONFIG_DESKTOP_INIT_LOG_HID_REPORT_EVENT=n
+CONFIG_DESKTOP_INIT_LOG_HID_REPORT_SENT_EVENT=n
+CONFIG_CAF_INIT_LOG_KEEP_ALIVE_EVENTS=n
+
+CONFIG_DESKTOP_ROLE_HID_PERIPHERAL=y
+CONFIG_DESKTOP_PERIPHERAL_TYPE_MOUSE=y
+CONFIG_DESKTOP_DEVICE_PID=0x52DE
+CONFIG_DESKTOP_HID_BOOT_INTERFACE_MOUSE=y
+
+CONFIG_DESKTOP_MOTION_BUTTONS_ENABLE=y
+CONFIG_DESKTOP_MOTION_BUTTONS_UP_KEY_ID=0
+CONFIG_DESKTOP_MOTION_BUTTONS_DOWN_KEY_ID=3
+CONFIG_DESKTOP_MOTION_BUTTONS_LEFT_KEY_ID=2
+CONFIG_DESKTOP_MOTION_BUTTONS_RIGHT_KEY_ID=1
+
+CONFIG_CAF_BUTTONS=y
+CONFIG_CAF_BUTTONS_POLARITY_INVERSED=y
+CONFIG_CAF_BUTTONS_PM_KEEP_ALIVE=n
+
+CONFIG_CAF_CLICK_DETECTOR=y
+
+CONFIG_CAF_LEDS=y
+
+CONFIG_CAF_FORCE_POWER_DOWN_EVENTS=y
+
+# The LLPM must be explicitly enabled, as the Bluetooth Controller
+# is not part of the main application.
+CONFIG_CAF_BLE_USE_LLPM=y
+
+CONFIG_DESKTOP_BLE_USE_DEFAULT_ID=y
+
+CONFIG_DESKTOP_BLE_PEER_CONTROL=y
+CONFIG_DESKTOP_BLE_PEER_CONTROL_BUTTON=0x0000
+CONFIG_DESKTOP_BLE_PEER_ERASE_ON_START=y
+
+CONFIG_DESKTOP_BLE_SECURITY_FAIL_TIMEOUT_S=10
+
+CONFIG_DESKTOP_BLE_LOW_LATENCY_LOCK=y
+
+CONFIG_DESKTOP_CONFIG_CHANNEL_ENABLE=y
+CONFIG_DESKTOP_CONFIG_CHANNEL_DFU_ENABLE=y
+
+# Power management is not supported on this target.
+CONFIG_DESKTOP_POWER_MANAGER=n
+
+################################################################################
+# Zephyr Configuration
+
+CONFIG_SYSTEM_WORKQUEUE_STACK_SIZE=3584
+CONFIG_ISR_STACK_SIZE=2560
+CONFIG_MAIN_STACK_SIZE=2816
+CONFIG_BT_RX_STACK_SIZE=2048
+CONFIG_BT_HCI_TX_STACK_SIZE_WITH_PROMPT=y
+CONFIG_BT_HCI_TX_STACK_SIZE=1536
+
+CONFIG_BOOT_BANNER=n
+
+CONFIG_NUM_COOP_PRIORITIES=10
+CONFIG_NUM_PREEMPT_PRIORITIES=11
+
+CONFIG_HEAP_MEM_POOL_SIZE=2560
+CONFIG_IDLE_STACK_SIZE=512
+
+CONFIG_HW_STACK_PROTECTION=y
+CONFIG_RESET_ON_FATAL_ERROR=n
+
+CONFIG_SPEED_OPTIMIZATIONS=y
+
+CONFIG_REBOOT=y
+
+CONFIG_GPIO=y
+
+CONFIG_LED=y
+# PWM not supported on this target, using GPIO instead.
```

Comment:
PWM seems already supported for nRF54H20. I would suggest to open a follow-up ticket to enable it.

Reply:
Have you opened the ticket already? Please link opened ticket in similar case to simplify finding it

Reply:
https://nordicsemi.atlassian.net/browse/NCSDK-27489